### PR TITLE
az-repos: add page

### PR DESCRIPTION
**Pull request by sannae**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

For: #6141

### CODE REVIEW

File: pages/common/az-repos.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# az repos
+
+> Manage Azure DevOps repos.
+> Part of `azure-cli`.
+> More information: <https://learn.microsoft.com/it-it/cli/azure/repos?view=azure-cli-latest>.
```
Comment (kbdharun) [inline pages/common/az-repos.md:5]:
```suggestion
> More information: <https://learn.microsoft.com/en-us/cli/azure/repos?view=azure-cli-latest>.
```
We generally prefer non-localized versions of Microsoft URLs.

File: pages/common/az-repos.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- List all repos in a specific project:
+
+`az repos list --project {{project_name}}`
+
+- Add policy on specific branch of a specific repository to restrict basic merge:
```
Comment (kbdharun) [inline pages/common/az-repos.md:11]:
```suggestion
- Add policy on a specific branch of a specific repository to restrict basic merge:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/az-repos.md (hunk 1 above)
Comment (sannae) [reply pages/common/az-repos.md:5]: 🤦🏻 My bad! I missed it, thank you for it

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**